source/configuration_manual/namespace/index.rst

```diff
@@ -143,24 +178,28 @@ Example::
 ``namespace/order``
 -------------------
 
- - Default: `0`
- - Values: :ref:`uint`
+- Default: ``0``
+- Values:  :ref:`uint`
 
 Sets display order in IMAP ``NAMESPACE`` command.
-Automatically numbered when untouched.
+
+Namespaces are automatically numbered if this setting does not exist.
 
 
 .. _setting-namespace_prefix:
 
 ``namespace/prefix``
 --------------------
 
- - Default: <empty>
- - Values: :ref:`string`
+- Default: <empty>
+- Values:  :ref:`string`
 
 Specifies prefix for namespace.
+
 Must end with :ref:`hierarchy separator <setting-namespace_separator>`.
 
+:ref:`Mail user variables <variables-mail_user>` can be used.
```

Comment:
Maybe we should have a :ref:\`var-string\` for these settings that support %variables?.. Or possibly with more details like "var-mail-string", "var-auth-string" which would more directly point to which variables exactly are supported. But yeah, not really related to these changes.

Reply:
I'll add this document idea to a task.